cEP-0005.md

```diff
@@ -96,15 +96,13 @@ from coalib.results.Result import Result
 from coalib.bearlib.aspects import Root
 
 
-class RedundancyBear(LocalBear):
-    # coala will run bears based on this metadata
-    LANGUAGES = {"C", "Python"}
-    # This bear is supposed to be able to fix unused imports...
-    FIX_ASPECTS = {Root.Redundancy.UnusedImport}
-    # ... and detect code clones.
-    DETECT_ASPECTS = {Root.Redundancy.Clone}
+# To define the aspects and languages for a bear using the bear metaclass
+class RedundancyBear(LocalBear, aspects={
+    'detect': [Root.Redundancy.Clone],
+    'fix': [Root.Redundancy.UnusedImport]},
+    languages: ['C', 'Python']):
```

Comment:
Little syntax error ;) Must be `languages=['C', 'Python']`. And IMO, it would overall look better and PEP8ier like:

```python
class RedundancyBear(LocalBear, aspects={
        'detect': [Root.Redundancy.Clone],
        'fix': [Root.Redundancy.UnusedImport],
}, languages=['C', 'Python']):
    ...
```